### PR TITLE
Polish Translations

### PR DESCRIPTION
**Pull request by OskarZyg**

### CODE REVIEW

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+velocity.error.already-connected=Jesteś juz polaczony z tym serwerem!
+velocity.error.already-connected-proxy=Jesteś juz polaczony z tym serverem proxy!
```
Comment (crejk) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties]: Jesteś już połączony z tym proxy!

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+velocity.command.server-current-server=Jesteś obecnie połączony z {0}.
+velocity.command.server-too-many=Skonfigurowano zbyt wiele serwerów. Użyj uzupełniania kart, aby wyświetlić wszystkie dostępne serwery.
+velocity.command.server-available=Dostępne serwery:
+velocity.command.server-tooltip-player-online={0} gracz online
+velocity.command.server-tooltip-players-online={0} gracze online
+velocity.command.server-tooltip-current-server=Obecnie połączony z tym serwerem
+velocity.command.server-tooltip-offer-connect-server=Kliknij, aby połączyć się z tym serwerem
+
+velocity.command.glist-player-singular=gracz
+velocity.command.glist-player-plural=gracze
```
Comment (crejk) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties]: graczy

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+velocity.command.reload-success=Konfiguracja Velocity została pomyślnie załadowana ponownie.
+velocity.command.reload-failure=Nie można ponownie załadować konfiguracji Velocity. Sprawdź konsolę, aby uzyskać więcej informacji.
+
+velocity.command.version-copyright=Copyright 2018-2021 {0}. {1} jest objęty licencją na warunkach GNU General Public License v3.
+
+velocity.command.no-plugins=Obecnie nie ma zainstalowanych żadnych wtyczek.
+velocity.command.plugins-list=Wtyczki: {0}
+velocity.command.plugin-tooltip-website=Strona internetowa: {0}
+velocity.command.plugin-tooltip-author=Autor: {0}
+velocity.command.plugin-tooltip-authors=Autory: {0}
```
Comment (crejk) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties]: Autorzy

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+velocity.error.already-connected=Jesteś juz polaczony z tym serwerem!
+velocity.error.already-connected-proxy=Jesteś juz polaczony z tym serverem proxy!
+velocity.error.already-connecting=Już próbujesz połączyć się z serwerem!
+velocity.error.cant-connect=Nie można się połączyć z {0}: {1}
+velocity.error.connecting-server-error=Nie można się połączyć z {0}. Spróbuj ponownie później.
+velocity.error.connected-server-error=Twoje połączenie z {0} napotkało problem.
+velocity.error.internal-server-connection-error=Wystąpił wewnętrzny błąd połączenia z serwerem.
+velocity.error.logging-in-too-fast=Logujesz się zbyt szybko, spróbuj ponownie później.
+velocity.error.online-mode-only=Nie jesteś zalogowany to Minecrafta. Jeśli jesteś już zalogowany, sprubuj restartować twoj klient.
```
Comment (crejk) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties]: spróbuj*
But for me "Na serwer mogą wejść osoby tylko z oryginalną kopią gry. Jeśli jesteś zalogowany na swoim koncie Minecraft, spróbuj zrestartować gre." sounds better.

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+velocity.error.already-connected=Jesteś juz polaczony z tym serwerem!
```
Comment (crejk) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties]: Jesteś już połączony z tym serwerem!

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+velocity.error.already-connected-proxy=Jesteś juz polaczony z tym serverem proxy!
+velocity.error.already-connecting=Już próbujesz połączyć się z serwerem!
+velocity.error.cant-connect=Nie można się połączyć z {0}: {1}
```
Comment (crejk) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties]: Nie udało się połączyć z {0}: {1}

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+velocity.error.connecting-server-error=Nie można się połączyć z {0}. Spróbuj ponownie później.
```
Comment (crejk) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties]: Nie udało ci się się połączyć z {0}. Spróbuj ponownie później.

**Review by crejk — CHANGES_REQUESTED** [review]

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+velocity.command.command-does-not-exist=To polecenie nie istnieje.
+
+velocity.command.players-only=Tylko gracze mogą uruchomić to polecenie.
+velocity.command.server-does-not-exist=Podany serwer {0} nie istnieje.
+
+velocity.command.server-current-server=Jesteś obecnie połączony z {0}.
+velocity.command.server-too-many=Skonfigurowano zbyt wiele serwerów. Użyj uzupełniania kart, aby wyświetlić wszystkie dostępne serwery.
+velocity.command.server-available=Dostępne serwery:
+velocity.command.server-tooltip-player-online={0} gracz online
+velocity.command.server-tooltip-players-online={0} gracze online
```
Comment (crejk) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties]: {0} graczy online

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+velocity.command.server-tooltip-current-server=Obecnie połączony z tym serwerem
+velocity.command.server-tooltip-offer-connect-server=Kliknij, aby połączyć się z tym serwerem
+
+velocity.command.glist-player-singular=gracz
+velocity.command.glist-player-plural=gracze
+velocity.command.glist-total-suffix= są obecnie połączone z serwerem proxy.
```
Comment (crejk) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties]: jest aktualnie połączonych z proxy.

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties (hunk 4 above)
Comment (OskarZyg) [reply proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties]: The original message ("You are not logged into your Minecraft account. If you are logged into your Minecraft account, try restarting your Minecraft client.") doesn't really say anything about having a paid copy of Minecraft. I am open to changing this if more people want it to be changed to include owning a copy.

Comment (astei) [reply proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties]: The implication is that you have a paid copy of Minecraft. That is precisely what we refer to when a user has online mode enabled.

Comment (OskarZyg) [reply proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties]: https://github.com/VelocityPowered/Velocity/pull/466/commits/1f81ff813696ae2f4fc9a09f9588b8ae1ae5a2b9

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+velocity.error.already-connected=Jesteś już połączony z tym serwerem!
+velocity.error.already-connected-proxy=Jesteś już połączony z tym proxy!
+velocity.error.already-connecting=Już próbujesz połączyć się z serwerem!
+velocity.error.cant-connect=Nie udało się połączyć z {0}: {1}
+velocity.error.connecting-server-error=Nie udało ci się się połączyć z {0}. Spróbuj ponownie później.
+velocity.error.connected-server-error=Twoje połączenie z {0} napotkało problem.
+velocity.error.internal-server-connection-error=Wystąpił wewnętrzny błąd połączenia z serwerem.
+velocity.error.logging-in-too-fast=Logujesz się zbyt szybko, spróbuj ponownie później.
+velocity.error.online-mode-only=Na serwer mogą wejść osoby tylko z oryginalną kopią gry. Jeśli jesteś zalogowany na swoim koncie Minecraft, spróbuj zrestartować gre.
+velocity.error.player-connection-error=Wystąpił błąd wewnętrzny w twojim połączeniu.
```
Comment (Prof-Bloodstone) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties]: (typo)
```suggestion
velocity.error.player-connection-error=Wystąpił błąd wewnętrzny w twoim połączeniu.
```

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+
+velocity.error.already-connected=Jesteś już połączony z tym serwerem!
+velocity.error.already-connected-proxy=Jesteś już połączony z tym proxy!
+velocity.error.already-connecting=Już próbujesz połączyć się z serwerem!
+velocity.error.cant-connect=Nie udało się połączyć z {0}: {1}
+velocity.error.connecting-server-error=Nie udało ci się się połączyć z {0}. Spróbuj ponownie później.
+velocity.error.connected-server-error=Twoje połączenie z {0} napotkało problem.
+velocity.error.internal-server-connection-error=Wystąpił wewnętrzny błąd połączenia z serwerem.
+velocity.error.logging-in-too-fast=Logujesz się zbyt szybko, spróbuj ponownie później.
+velocity.error.online-mode-only=Na serwer mogą wejść osoby tylko z oryginalną kopią gry. Jeśli jesteś zalogowany na swoim koncie Minecraft, spróbuj zrestartować gre.
```
Comment (Prof-Bloodstone) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties]: (typo)
```suggestion
velocity.error.online-mode-only=Na serwer mogą wejść osoby tylko z oryginalną kopią gry. Jeśli jesteś zalogowany na swoim koncie Minecraft, spróbuj zrestartować grę.
```

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+velocity.error.already-connected=Jesteś już połączony z tym serwerem!
+velocity.error.already-connected-proxy=Jesteś już połączony z tym proxy!
```
Comment (Prof-Bloodstone) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties]: (minor) I believe this sounds better/more correct, roommate agrees :)
The same way you have below `Już próbujesz ...`
```suggestion
velocity.error.already-connected=Już jesteś połączony z tym serwerem!
velocity.error.already-connected-proxy=Już jesteś połączony z tym proxy!
```

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties [hunk 13]
```diff
@@ -0,0 +1,62 @@
...
+velocity.error.already-connecting=Już próbujesz połączyć się z serwerem!
+velocity.error.cant-connect=Nie udało się połączyć z {0}: {1}
+velocity.error.connecting-server-error=Nie udało ci się się połączyć z {0}. Spróbuj ponownie później.
```
Comment (Prof-Bloodstone) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties]: When addressing someone, use upper case. Also don't repeat yourself :)
Also reworded slightly to be more in line with English version.
```suggestion
velocity.error.connecting-server-error=Nie udało się Ciebie połączyć z {0}. Spróbuj ponownie później.
```

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties [hunk 14]
```diff
@@ -0,0 +1,62 @@
...
+velocity.error.connected-server-error=Twoje połączenie z {0} napotkało problem.
+velocity.error.internal-server-connection-error=Wystąpił wewnętrzny błąd połączenia z serwerem.
+velocity.error.logging-in-too-fast=Logujesz się zbyt szybko, spróbuj ponownie później.
+velocity.error.online-mode-only=Na serwer mogą wejść osoby tylko z oryginalną kopią gry. Jeśli jesteś zalogowany na swoim koncie Minecraft, spróbuj zrestartować gre.
+velocity.error.player-connection-error=Wystąpił błąd wewnętrzny w twojim połączeniu.
+velocity.error.modern-forwarding-needs-new-client=Ten serwer jest kompatybilny tylko z Minecraft 1.13 i nowszymi wersjami.
+velocity.error.modern-forwarding-failed=Twój serwer nie wysłał żądania przekazania do serwera proxy. Upewnij się, że serwer jest skonfigurowany do przesyłania Velocity.
```
Comment (Prof-Bloodstone) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties]: (minor) I believe `przekazywać` is a better word in this context, than `przesyłać`.
```suggestion
velocity.error.modern-forwarding-failed=Twój serwer nie wysłał żądania przekazania do serwera proxy. Upewnij się, że serwer jest skonfigurowany do przekazywania Velocity.
```

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties [hunk 15]
```diff
@@ -0,0 +1,62 @@
...
+velocity.error.moved-to-new-server=Zostałeś wyrzucony z {0}: {1}
+velocity.error.no-available-servers=Brak dostępnych serwerów z którymi można by się połączyć. Spróbuj ponownie później lub skontaktuj się z administratorem.
```
Comment (Prof-Bloodstone) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties]: (minor) You are connecting someone, not yourself (see English version)
```suggestion
velocity.error.no-available-servers=Brak dostępnych serwerów z którymi można by Ciebie połączyć. Spróbuj ponownie później lub skontaktuj się z administratorem.
```

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties [hunk 16]
```diff
@@ -0,0 +1,62 @@
...
+
+# Commands
+velocity.command.generic-error=Wystąpił błąd podczas uruchamiania tego polecenia.
```
Comment (Prof-Bloodstone) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties]: (minor)
```suggestion
velocity.command.generic-error=Wystąpił błąd podczas wykonywania tego polecenia. 
```

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties [hunk 17]
```diff
@@ -0,0 +1,62 @@
...
+
+# Commands
+velocity.command.generic-error=Wystąpił błąd podczas uruchamiania tego polecenia. 
+velocity.command.command-does-not-exist=To polecenie nie istnieje.
+
+velocity.command.players-only=Tylko gracze mogą uruchomić to polecenie.
+velocity.command.server-does-not-exist=Podany serwer {0} nie istnieje.
+
+velocity.command.server-current-server=Jesteś obecnie połączony z {0}.
+velocity.command.server-too-many=Skonfigurowano zbyt wiele serwerów. Użyj uzupełniania kart, aby wyświetlić wszystkie dostępne serwery.
```
Comment (Prof-Bloodstone) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties]: For sure we are not completing any cards here. I'd change the beginning of second sentence to either of these:

- `Użyj uzupełniania tabem` (might be good to accent TAB by writing `TABem`)
- `Użyj uzupełniania klawiszem tab`

**Review by Prof-Bloodstone — CHANGES_REQUESTED** [review]
A few changes :)

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties [hunk 18]
```diff
@@ -0,0 +1,62 @@
...
+velocity.command.server-available=Dostępne serwery:
+velocity.command.server-tooltip-player-online={0} gracz online
+velocity.command.server-tooltip-players-online={0} graczy online
+velocity.command.server-tooltip-current-server=Obecnie połączony z tym serwerem
+velocity.command.server-tooltip-offer-connect-server=Kliknij, aby połączyć się z tym serwerem
+
+velocity.command.glist-player-singular=gracz
+velocity.command.glist-player-plural=graczy
+velocity.command.glist-total-suffix= jest aktualnie połączonych z proxy.
```
Comment (Prof-Bloodstone) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties]: As per <https://github.com/VelocityPowered/Velocity/commit/d26d61f0389e922317a76e749f85ebb8edc8045f>
```suggestion
velocity.command.glist-player-singular={0} gracz jest aktualnie połączony z proxy.
velocity.command.glist-player-plural={0} graczy jest aktualnie połączonych z proxy.
```

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties (hunk 12 above)
Comment (CDFN) [reply proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties]: I disagree on this. `Jesteś już` sounds more natural to me.

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties (hunk 17 above)
Comment (Prof-Bloodstone) [reply proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties]: Maybe `Użyj uzupełniania tabulatorem`?

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties (hunk 12 above)
Comment (OskarZyg) [reply proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties]: At first, I felt "Jesteś już" was more natural but after some consideration, I think "Już jesteś" sounds a bit better.

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties [hunk 19]
```diff
@@ -15,39 +15,38 @@
...
 #
 
-velocity.error.already-connected=Jesteś już połączony z tym serwerem!
-velocity.error.already-connected-proxy=Jesteś już połączony z tym proxy!
+velocity.error.already-connected=Już jesteś połączony z tym serwerem!
+velocity.error.already-connected-proxy=Już jesteś połączony z tym proxy!
 velocity.error.already-connecting=Już próbujesz połączyć się z serwerem!
 velocity.error.cant-connect=Nie udało się połączyć z {0}: {1}
-velocity.error.connecting-server-error=Nie udało ci się się połączyć z {0}. Spróbuj ponownie później.
+velocity.error.connecting-server-error=Nie udało Ciebie połączyć z {0}. Spróbuj ponownie później.
```
Comment (Prof-Bloodstone) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties]: You deleted too many `się` :P
```suggestion
velocity.error.connecting-server-error=Nie udało się Ciebie połączyć z {0}. Spróbuj ponownie później.
```

**Review by Prof-Bloodstone — APPROVED** [review]
Apart from that small mistake, LGTM :)

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties [hunk 20]
```diff
@@ -0,0 +1,61 @@
...
+
+velocity.error.already-connected=Już jesteś połączony z tym serwerem!
+velocity.error.already-connected-proxy=Już jesteś połączony z tym proxy!
+velocity.error.already-connecting=Już próbujesz połączyć się z serwerem!
+velocity.error.cant-connect=Nie udało się połączyć z {0}: {1}
+velocity.error.connecting-server-error=Nie udało się Ciebie połączyć z {0}. Spróbuj ponownie później.
+velocity.error.connected-server-error=Twoje połączenie z {0} napotkało problem.
+velocity.error.internal-server-connection-error=Wystąpił wewnętrzny błąd połączenia z serwerem.
+velocity.error.logging-in-too-fast=Logujesz się zbyt szybko, spróbuj ponownie później.
+velocity.error.online-mode-only=Na serwer mogą wejść osoby tylko z oryginalną kopią gry. Jeśli jesteś zalogowany na swoim koncie Minecraft, spróbuj zrestartować gré.
```
Comment (crejk) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties]: grę*

**Comment by astei** [comment]
Can you update your translations with the new messages in 1e8454c7? Thanks!

**Comment by OskarZyg** [comment]
I'm not exactly sure what word to put on line 65 for "developer", right now I've chosen `programista`. I also replaced the word dump with upload in `dump-server-error` since it didn't sound very natural.
I am open to changes about these, of course.

**Comment by astei** [comment]
[Imported into Crowdin](https://crowdin.com/project/velocity/pl)

**Pull request closed without merging**